docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+# Support Bundle User Guide
+
+## What is Supportbundle
+
+Antrea supports creating support bundle tarballs, which include information from the cluster,
+the Antrea Controller, Antrea Agents and ExternalNode managed by the cluster.
+The information is useful when troubleshooting issues in Kubernetes clusters using Antrea.
+
+**Be aware that the generated support bundle includes a lot of information,
+  including logs, so please review the contents of the directory before sharing
+  it on Github and ensure that you do not share anything sensitive.**
+
+There are two ways of generating support bundles. Firstly, You can run `antctl supportbundle`
+directly in the Antrea Agent Pod, Antrea Controller Pod, or on the Host
+with `kubeconfig` targeting Antrea Cluster. Secondly, You can also apply
+SupportBundleCollection YAMLs to create support bundles for Kubernetes Nodes or external Nodes.
+We use "SupportBundle Collection" to designate such feature. The details will be provided
+in section [Use case example](#use-case-example).
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Use case example](#use-case-example)
+  - [Running antctl commands](#running-antctl-commands)
+  - [Applying SupportBundleCollection CRD](#applying-supportbundlecollection-crd)
+- [List of collected items](#list-of-collected-items)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Prerequisites
+
+SupportBundle Collection is introduced in Antrea v1.10.0 as an alpha feature. The feature gate
+`SupportBundleCollection` must be enabled in the `antrea-agent` configuration to use the feature.
```

Comment:
The feature gate must be enabled in both antrea-controller and antrea-agent configurations

Reply:
Remove "to use the feature"?